Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -266,6 +266,8 @@ - (NSString *)reCAPTCHATokenForURL:(NSURL *)URL error:(NSError **)error {
     NSString *reason;
     if(errorDict[@"code"] && errorDict[@"message"]) {
       reason = [NSString stringWithFormat:@"[%@] - %@",errorDict[@"code"], errorDict[@"message"]];
+    } else {
+      reason = @"An unknown error has occurred";
```

Comment:
On second thought, we can append the deep link string to the end of this message;

`[NSString StringWithFormat:@"An unknown error occurred with the following response:  %@", deepLinkURL]`